pkg/k8s/application/k8sApplicationService.go

```diff
@@ -688,13 +691,15 @@ func (impl *K8sApplicationServiceImpl) GetResourceList(ctx context.Context, toke
 		impl.logger.Errorw("error on parsing for k8s resource", "err", err)
 		return resourceList, err
 	}
-	k8sServerVersion, err := impl.k8sCommonService.GetK8sServerVersion(clusterId)
-	if err != nil {
-		impl.logger.Errorw("error in getting k8s server version", "clusterId", clusterId, "err", err)
-		// return nil, err
-	} else {
-		resourceList.ServerVersion = k8sServerVersion.String()
-	}
+	// Not used in FE side
```

Comment:
I think this is used for ephemeral containers support in FE. please reconfirm

Reply:
verified not getting used in this API